PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -6,6 +6,51 @@ namespace PrincessBrideTrivia.Tests;
 public class ProgramTests
 {
     [TestMethod]
+public void IncorrectQuestions_AreRecordedWhenAnsweredWrong()
+{
+    // Arrange
+    Question question1 = new()
```

Comment:
```suggestion
    Question question = new()
```
nit: no need to add a 1 if there is only one instance of question in your scope